api/src/data_migration/command/load_transform.py

```diff
@@ -32,6 +33,9 @@
     "--insert-chunk-size", default=800, help="chunk size for load inserts", show_default=True
 )
 @click.option("--tables-to-load", "-t", help="table to load", multiple=True)
+@click.option(
+    "--store-version/--no-store-version", default=True, help="run StoreOpportunityVersionTask"
+)
```

Comment:
Let us default this to not being enabled, I think we should test the behavior out manually first to be safe.